wadsrc/static/zscript/ui/statusbar/doom_sbar.zs

```diff
@@ -89,21 +89,19 @@ class DoomStatusBar : BaseStatusBar
 		String image;
 		for(int i = 0; i < 6; i++) locks[i] = CPlayer.mo.CheckKeys(i + 1, false, true);
 		// key 1
-		if (locks[1] && locks[4]) image = "STKEYS6";
+		if (locks[1] && locks[4]) image = "ZTKEYS1";
 		else if (locks[1]) image = "STKEYS0";
 		else if (locks[4]) image = "STKEYS3";
 		DrawImage(image, (239, 171), DI_ITEM_OFFSETS);
 		// key 2
-		if (locks[2] && locks[5]) image = "STKEYS7";
+		if (locks[2] && locks[5]) image = "ZTKEYS2";
 		else if (locks[2]) image = "STKEYS1";
 		else if (locks[5]) image = "STKEYS4";
-		else image = "";
```

Comment:
Removing these "else image" lines is wrong, also. That is simply going to cause the top key to be repeated twice if the other two keys have not yet been gotten.

Reply:
imo it'd be wise to put them in all of the if statements relating to the drawing of those keys to prevent this. Will fix shortly.

Reply:
Fixed.